core/tauri/permissions/app/autogenerated/reference.md

```diff
@@ -4,6 +4,10 @@
 |`deny-app-hide`|Denies the app_hide command without any pre-configured scope.|
 |`allow-app-show`|Enables the app_show command without any pre-configured scope.|
 |`deny-app-show`|Denies the app_show command without any pre-configured scope.|
+|`allow-default-window-icon`|Enables the default_window_icon command without any pre-configured scope.|
+|`deny-default-window-icon`|Denies the default_window_icon command without any pre-configured scope.|
+|`allow-get-default-window-icon`|Enables the get_default_window_icon command without any pre-configured scope.|
+|`deny-get-default-window-icon`|Denies the get_default_window_icon command without any pre-configured scope.|
```

Comment:
Is this a leftover or is our generator broken?

Reply:
this is a leftover from a previous iteration, I will regenerate